styleguide/_themes/global/scss/fonts.scss

```diff
@@ -13,10 +13,11 @@ $code: Tahoma, Arial, Geneva, Helvetica;
 @mixin subhead($color: $gray-darkest) {
   color: $color;
   font-family: $subhead;
-  font-weight: 400;
+  font-weight: 500;
+  letter-spacing: .35px;
```

Comment:
I'm not sure px are the appropriate unit here. Probably `em` would be best so it's always relative to the font size.